packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -76,42 +78,40 @@ const DomainPickerSuggestionItem: React.FC< Props > = ( {
 	const [ previousDomain, setPreviousDomain ] = React.useState< string | undefined >();
 	const [ previousRailcarId, setPreviousRailcarId ] = React.useState< string | undefined >();
 
+	// translators: 'Default' will be shown next to the standard, free domain
 	const freeDomainLabelDefault = __( 'Default', __i18n_text_domain__ );
 	const freeDomainLabelFree = __( 'Free', __i18n_text_domain__ );
+	const freeDomainLabel =
+		type === ITEM_TYPE_INDIVIDUAL_ITEM ? freeDomainLabelDefault : freeDomainLabelFree;
+
+	const fallbackIncludedLabel = __( 'Included with annual plans', __i18n_text_domain__ );
+	const newIncludedLabel = __( 'Included in annual plans', __i18n_text_domain__ );
+	const includedLabel =
+		locale === 'en' || hasTranslation?.( 'Included in annual plans', __i18n_text_domain__ )
+			? newIncludedLabel
+			: fallbackIncludedLabel;
 
-	const firstYearLabel = __( 'Included in paid plans', __i18n_text_domain__ );
-	const firstYearLabelAlt = __( 'Included with annual plans', __i18n_text_domain__ );
 	// translators: text in between the <strong></strong> marks is styled as bold text
-	const firstYearLabelFormatted = __(
+	const fallbackIncludedLabelFormatted = __(
 		'<strong>First year included</strong> in paid plans',
 		__i18n_text_domain__
 	);
-
-	const freeDomainLabel =
-		type === ITEM_TYPE_INDIVIDUAL_ITEM ? freeDomainLabelDefault : freeDomainLabelFree;
-
-	const firstYearIncludedInPaidLabel = isMobile
-		? firstYearLabel
-		: createInterpolateElement( firstYearLabelFormatted, {
+	// translators: text in between the <strong></strong> marks is styled as bold text
+	const newIncludedLabelFormatted = __(
+		'<strong>First year included</strong> in annual plans',
+		__i18n_text_domain__
+	);
+	const includedLabelFormatted =
+		locale === 'en' ||
+		hasTranslation?.( '<strong>First year included</strong> in annual plans', __i18n_text_domain__ )
```

Comment:
Hey @Automattic/i18n , I've noticed that this check seems to return false in Gutenboarding even if the strings should be available — do you know why this is happening?

As a comparison, a similar check [a few lines above (line 90)](https://github.com/Automattic/wp-calypso/pull/49290/files#diff-df652cccef87aecec53f79cf4fd560f677b5af1c74439cfb7a7bbbff2da584c3R90) seems to be working fine in my tests.

As an additional check, on my local machine I tried removing the second argument to this call — and `hasTranslation?.( '<strong>First year included</strong> in annual plans' )` seems to work.

Any clues on why this is happening?

, when we pass the context in this `hasTranslation` call, the result is `false` even though the translations are in.

I tested removing the second argument and only calling `hasTranslation?.( '<strong>First year included</strong> in annual plans')`, and that seems to work (there's a match in Gutenboarding).

Is this behaviour expected, or is it a bug? When imported in Gutenboarding, the `__i18n_text_domain__` constant is `default`,

Reply:
Just pasting what I'm seeing when trying to debug `hasTranslation@packages/react-i18n/src/index.tsx`.

```
function hasTranslation(localeData, singular, context) {
  const key = typeof context === 'string' ? ''.concat(context, CONTEXT_DELIMETER, singular) : singular;
  return key in localeData;
}
```

![image](https://user-images.githubusercontent.com/1287077/109691221-83627000-7b87-11eb-80d5-aec391c70027.png)

Reply:
Perhaps we can just check for the translation without the context, e.g.
```
hasTranslation?.( '<strong>First year included</strong> in annual plans' )
```

That kinda fits the logic written in `hasTranslation`.

Reply:
The problem is that you're passing `__i18n_text_domain__` as a second parameter to `hasTranslation` where it expects `context`, and those two are different things. Given that translations in Calypso are stored in a single domain (applies for `i18n-calypso` and `@automattic/react-i18n`), I think it's fine for now to use `hasTranslation` with just the string as only parameter.

Reply:
Thank you @yuliyan — I confused `context` and `domain` 😅 I will remove the second parameter to the `hasTranslation` calls

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/49290/commits/bc4e2c9ef8b27c07ca7425448f9f950687dff1d9